### DISCUSSION TITLE
How to edit the wiki?

### DISCUSSION THREAD

**Discussion by Landorin-GH** [AdguardTeam/AdGuardHome#7336] [Q&A]
Hi all,

sorry for this lame question but how do I edit the Adguard Home wiki in the browser on github? I had it on my todo list for a long time but only saw some option to clone it locally which is overkill. I would like to update the dns encryption page especially as that always has been quite a misunderstanding for quite some users. The referred knowledge base website does not host this information at all and is therefore not an option really.

**Comment by Dynamic5912** [comment]
I believe only admins or assigned users can do this..
